monai/inferers/inferer.py

```diff
@@ -120,7 +121,13 @@ def __init__(
                 )
         self.splitter = splitter
 
-        # mergers
+        # merger
+        if isinstance(merger_cls, str):
+            valid_merger: type[Merger]
+            valid_merger, merger_found = optional_import("monai.inferers.merger", name=merger_cls)  # search mergers
+            if not merger_found:
+                raise ValueError(f"The requested merger ['{merger_cls}'] does not exist.")
```

Comment:
Could you please help add the support for user customized `Merger`? Refer to:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/transforms/io/array.py#L206

Thanks.

Reply:
Sure